src/scripts/components/Html.js

```diff
@@ -20,6 +20,11 @@ class Html extends React.Component {
           <meta property="og:url" content="http://quran.com/" />
           <meta property="og:image" content="http://quran.com/images/thumbnail.png" />
 
+          <meta name="twitter:card" content="summary_large_image" />
+          <meta name="twitter:site" content="@nytimes" />
```

Comment:
need to replace this with an account - if we don't have one still, use QuranAndroid for now maybe?